Lesson_2_JSON_YAML_XML/lesson2_3.xml

```diff
@@ -0,0 +1,28 @@
+<cars>
+  <car>
+    <make>Lamborghini</make>
+    <model>Aventador</model>
+    <year>2021</year>
+    <isElectric>false</isElectric>
+    <availableColors>
+      <color>Yellow</color>
```

Comment:
I love this nested structure of a parent element with multiple children as elements of an array, well done!